src/textures/parsers/SpriteSheet.js

```diff
@@ -58,6 +58,11 @@ var SpriteSheet = function (texture, sourceIndex, x, y, width, height, config)
     var column = Math.floor((height - margin + spacing) / (frameHeight + spacing));
     var total = row * column;
 
+    if(total === 0)
+    {
+        console.warn('TextureManager.SpriteSheet: Frame config produces zero frames. Check frameWidth and frameHeight.');
```

Comment:
I'm happy with this addition, but I'd like to make the warning shorter as it doesn't get minified in the production bundle. Perhaps we could convey the same message with fewer words?